ServerCore/Pages/Teams/Apply.cshtml.cs

```diff
@@ -74,6 +74,12 @@ public async Task<IActionResult> OnGet(int teamID)
 
             await _context.SaveChangesAsync();
 
+            // TODO I am well aware that it would be far better to include a direct link to the details page,
+            // but I cannot for the life of me figure out how to generate a URL for a Razor page.
+            MailHelper.Singleton.SendPlaintextWithoutBcc(new string[] { Team.PrimaryContactEmail, LoggedInUser.Email },
```

Comment:
Do you need to check if the PrimaryContactEmail is null first? Will it just bounce if that's the case?

Reply:
We should have a separate work item to disallow null primary contacts.

Reply:
The mail code will drop any invalid email address including null.

Reply:
I just realized mail code will probably get an exception if all the email addresses are invalid and you're not doing bcc. But since we use SendMailAsync without an async method or callback, I think that exception will just get dropped.

Reply:
Agreed but for the applicant's benefit we should make sure the team is configured to see the request. This code is safe to check in before we do that part, thanks to your error handling code.

Reply:
Ditto the player's email, we should make that required as well if it isn't already.

Reply:
Yes. Here's the regex I think we should validate against
`/[^@\s]+@[.A-Za-z0-9-]+(\s+[,;]\s+[^@\s]+@[.A-Za-z0-9-]+)/`.
There are lots of invalid email addresses this will match and this errs on the side of avoiding false negatives vs. avoiding false positives.